python-aexpect.spec

```diff
@@ -20,6 +20,9 @@
 %global gittar		%{srcname}-%{shortcommit}.tar.gz
 %endif
 
+# Selftests are provided but skipped because they use unsupported tooling.
+%global with_tests 0
```

Comment:
OK, because inspector is not available the default is 0, do I understand it correctly? Let's hope it'll be added soon so we can enable them.

Reply:
Yes, that why we have https://trello.com/c/kZbGSE10/1279-inspektor-add-rpm-packaging